git reset --hard

```diff
@@ -0,0 +1,17 @@
+[33mcommit 3ea9b4fc03d0bec31e64d950365cf180449de782[m[33m ([m[1;36mHEAD -> [m[1;32mmyFeatureBranch[m[33m, [m[1;32mmain[m[33m)[m
```

Comment:
Looks like the output of `git log` somehow became a file and got added to this commit, can you remove it?